sdks/wasm/sdk/Mono.WebAssembly.Build/RuntimeJs.tt

```diff
@@ -14,10 +29,19 @@ var Module = {
 	},
 };
 
-
+(function() {
```

Comment:
nit: space before parenths (and a bit below)

Reply:
Maxim not sure I see this one.

Reply:
need to be added :)